src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card">
+        <MovieCard
```

Comment:
on the MovieCard component, you already add div with the class card. So on results you have:
```
<div className="card">
  <div className="card">
    ...
  </div>
</div>
```